documentation/docs/30-add-ons/25-paraglide.md

```diff
@@ -0,0 +1,38 @@
+---
+title: paraglide
+---
+
+[Paraglide from Inlang](https://inlang.com/m/gerre34r/library-inlang-paraglideJs) is a compiler-based i18n library that emits tree-shakable message functions with small bundle sizes, no async waterfalls, full type-safety, and more.
+
+## Usage
+
+```bash
+npx sv add paraglide
+```
+
+## What you get
+
+- Inlang project settings
+- paraglide Vite plugin
+- SvelteKit `reroute` and `handle` hooks
+- `text-direction` and `lang` attributes in `app.html`
+- updated `.gitignore`
+- an optional demo page showing how to use paraglide
+
+## Options
+
+### availableLanguageTags
+
+The languages you'd like to support specified as IETF BCP 47 language tags.
+
+```bash
+npx sv add --availableLanguageTags=en,es
```

Comment:
```suggestion
npx sv add --paraglide==en,es
```
That does not actually work as pointed out by @AdrianGonz97 in #543 but it's more correct. We will need to adjust this once #543 lands, so I don't think this should be a problem.

Reply:
Did you mean to suggest two equals signs here?

Reply:
Absolutely not!